roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -29,6 +29,17 @@
   failed_when: "'already exists' not in osmiq_create_service_account.stderr and osmiq_create_service_account.rc != 0"
   changed_when: osmiq_create_service_account.rc == 0
 
+- name: Create Image Scanner Service Account
```

Comment:
I'd personally be okay with miq-<name> instead of management-<name> management almost seems like it is a built in SA and not an integration.